requirements.txt

```diff
@@ -9,6 +9,6 @@ astropy>=3.2.1
 corner
 pymc3
 hdbscan
-sklearn
+scikit-learn<=0.22.0
```

Comment:
Should probably keep an eye on scikit-learn updates so that if/when the macos bug is fixed we can change this to `>=` some version.